test/integration/visitor_can_add_items_to_cart_test.rb

```diff
@@ -0,0 +1,46 @@
+require 'test_helper'
+
+class VisitorCanAddItemsToCart < ActionDispatch::IntegrationTest
+
+  test "visitor can add items to cart from index" do
+    category_1 = OilType.create(name: "Lard")
+    category_2 = OilType.create(name: "Coconut Oil")
+
+    item_1 = Chip.create(name: "Slotachips", price: 20, description: "Super yummy", oil_type_id: category_1.id)
+    item_2 = Chip.create(name: "Trader Joe's BBQ", price: 15, description: "I'd trade slota for these!", oil_type_id: category_2.id)
+    item_3 = Chip.create(name: "Dang Coconut", price: 17, description: "Dang, these are good", oil_type_id: category_2.id)
```

Comment:
Useless assignment to variable - `item_3`. Did you mean `item_1`?<br>Line is too long. [122/80]